lib/broadside/deploy/ecs_deploy.rb

```diff
@@ -73,8 +76,16 @@ def bootstrap
         info "Service '#{family}' doesn't exist, creating..."
         EcsManager.create_service(config.ecs.cluster, family, @deploy_config.service_config)
       end
+    end
 
-      @deploy_config.bootstrap_commands.each { |command| run_command(command) }
+    def run_bootstrap_commands(update_task_def: false)
+      update_task_revision if update_task_def
```

Comment:
wouldn't it be `update_task_def = false` for a default arg?

also i don't know if it's worth the complication to ever _not_ update the task definition when bootstrapping or deploying... certainly when deploying we just update it no matter what and @matl33t had said that was fine, given the nature of task definition revisions.